pytentiostat/main.py

```diff
@@ -24,39 +24,39 @@ def configure_board(self):
     board_instance = BoardCom()
     # com, board, a0, a2, d9 = startup_routine()
 except KeyboardInterrupt:
-    sys.exit("Stopped. Make any changes and then restart the software.\nExiting...")
+    sys.exit("Connection interrupted. Make sure the potentiostat is plugged in then restart the software.\n"
+             "Exiting...")
 
 board_objects = (board_instance.pin_a0, board_instance.pin_a2, board_instance.pin_d9)
 
-reconfig = "sting"
+reconfig = "string"
 
 while True:
     try:
         # Run the experiment and get the config_data
         times, voltages, currents = experiment(config_data, *board_objects)
         break
     except KeyboardInterrupt:
-        reconfig = input("Experiment stopped. To restart the same experiment, press Enter. \n"
-                         "If you want to do a different experiment, type \"New\"\n"
+        reconfig = input("Experiment interrupted. To restart the same experiment, press Enter. \n"
+                         "If you want to do a different experiment, edit and save the config file then type"
+                         " \"new\" and press enter\n"
                          "If you need to reconnect the poteniostat, type \"Reconnect\" \n")
+        reconfig = verify_input(reconfig)
         if reconfig == "":
             reconfig = "string"
             plt.close()
-            continue
         elif reconfig.lower() == "new":
             reconfig = "string"
             plt.close()
             config_data = parse_config_file()
-            continue
         elif reconfig.lower() == "reconnect":
             reconfig = "string"
             plt.close()
             closing_routine(board_instance.board, board_instance.pin_d9)
             board_instance.configure_board()
             board_objects = (board_instance.pin_a0, board_instance.pin_a2, board_instance.pin_d9)
-            continue
-        elif reconfig == "string":
-            break
+        else:
+            sys.exit("Value error. Exiting...")
```

Comment:
Maybe we can discuss this at the next meeting. Once the GUI is out, of course we won't need any pauses but now it think a few are helpful just to prevent the program from executing everything at once. I also think it allows the user to tune the experiment a little on the fly which can be helpful.